src/pipx/commands/install.py

```diff
@@ -102,6 +107,22 @@ def install(
                 include_dependencies,
                 force=force,
             )
+        except FileNotFoundError as e:
+            venv.remove_venv()
+            if python in str(e) or "The system cannot find the file specified" in str(e):
+                print(
+                    pipx_wrap(
+                        f"""
+                           No executable for the provided Python version '{python}' found.
+                           Please make sure the executable name is on your PATH /
+                           the path to the executable is correct.
+                        """
+                    )
```

Comment:
This could make it look nicer. Could you also import `pipx.emojis.hazard` and insert it?

```suggestion
                    pipx_wrap(
                        f"""
                        No executable for the provided Python version '{python}' found.
                        Please make sure the executable name is on your PATH /
                        the path to the executable is correct.
                        """,
                        subsequent_indent=" " * 4,
                    )
```

Reply:
Added your suggestions